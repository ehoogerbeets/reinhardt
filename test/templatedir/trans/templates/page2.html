{% blocktrans with foo=page.name %}
This is the second block trans with multiple
lines in the source and a renamed variable in it.
Here is the page name: {{foo}} 
{% endblocktrans %}
