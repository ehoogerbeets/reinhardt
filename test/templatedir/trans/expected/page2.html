 Dies ist der zweite trans-Block mit mehreren Linien in der Quelle und einem umbenannten Variable darin. Hier ist der Name der Seite: Test page 
